### PR TITLE
{tools}[GCCcore/12.2.0] Emacs v28.2

### PR DESCRIPTION
**Pull request by eylenth**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by eylenth** [comment]
Test report by @eylenth
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
by0owl.eu.seeds.basf.net - Linux RHEL 7.9, x86_64, Intel(R) Xeon(R) CPU E5-2620 v4 @ 2.10GHz (broadwell), Python 3.6.8
See https://gist.github.com/eylenth/2bb17f1fa5df7aaa9c71a210f87247ee for a full test report.

File: easybuild/easyconfigs/e/Emacs/Emacs-28.2-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['a6912b14ef4abb1edab7f88191bfd61c3edd7085e084de960a4f86485cb7cad8']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('pkg-config', '0.29.2'),
```
Comment (Micket) [inline easybuild/easyconfigs/e/Emacs/Emacs-28.2-GCCcore-12.2.0.eb:27]:
```suggestion
    ('pkgconf', '1.9.3'),
```

We are trying to move away from pkg-config to the (actually maintained) pkgconf instead.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
himem04.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6230 CPU @ 2.10GHz (cascadelake), Python 3.6.8
See https://gist.github.com/jfgrimm/2f1392885edce3bfc9f13fa5bba98544 for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.7, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/4cb3091c6c6031b4a0be0302ed5e42fb for a full test report.

**Comment by Micket** [comment]
Going in, thanks @eylenth!

**Pull request merged**